src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -692,9 +692,11 @@ pal::string_t resolve_sdk_version(pal::string_t sdk_path, bool parse_only_produc
         if (fx_ver_t::parse(version, &ver, parse_only_production))
         {
             if (global_cli_version.empty() ||
-                // Pick the greatest version that differs only in the 'minor-patch' if a global cli version is specified.
+				// If a global cli version is specified:
+				//   pick the greatest version that differs only in the 'minor-patch'
+				//   and is semantically greater than or equal to the global cli version specified.
                 (ver.get_major() == specified.get_major() && ver.get_minor() == specified.get_minor() &&
-                (ver.get_patch() / 100) == (specified.get_patch() / 100)))
+                (ver.get_patch() / 100) == (specified.get_patch() / 100) && (ver == std::max(ver, specified))))
```

Comment:
Wouldn't `ver >= specified` be easier to understand?

Reply:
@nguerrera 
I'm not sure that works correctly...
'2.1.3' is semantically greater than '2.1.3-preview1'
but is '2.1.3' > '2.1.3-preview1'?

Reply:
If the comparison operators on `fx_ver_t` don't do what you want, then I don't understand how `std::max` would fare any better.

`std::max(a, b)` is basically `a < b ? b : a` 

Note also that there are other cases of ver >= specified in this file.

Reply:
@nguerrera 
Changed...